src/models/owner.rs

```diff
@@ -22,6 +23,21 @@ pub struct CrateOwner {
     pub email_notifications: bool,
 }
 
+type BoxedQuery<'a> = crate_owners::BoxedQuery<'a, Pg, crate_owners::SqlType>;
+
+impl CrateOwner {
+    /// Returns a base crate owner query filtered by the owner kind argument. This query also
+    /// filters out deleted records.
+    pub fn by_owner_kind(kind: OwnerKind) -> BoxedQuery<'static> {
+        use self::crate_owners::dsl::*;
+
+        crate_owners
+            .filter(deleted.eq(false))
+            .filter(owner_kind.eq(kind as i32))
```

Comment:
Oh this is so much nicer!! I love that this function encapsulates the casting of `OwnerKind` too :)